### ISSUE TITLE
Problem entering text in Angular2 TodoMVC tests

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#646]
This is coming from https://github.com/bahmutov/effective-e2e-testing-with-cypress/issues/2

- Cypress 0.19.4 and 0.20.0

Cannot type "enter" from the test into Angular2 example

```js
// Angular2
const url = 'http://todomvc.com/examples/angular2/'
context('New Todo', () => {
  beforeEach(() => {
    cy.visit(url)
  })

  const firstTodo = 'Learn Cypress'

  it('adds 1 todo', () => {
    cy.get('.new-todo').type(firstTodo).type('{enter}')
    cy.get('.todo-list li')
      .first()
      .find('label').should('contain', firstTodo)
  })
})
```

<img width="840" alt="screen shot 2017-09-11 at 4 47 24 pm" src="https://user-images.githubusercontent.com/2212006/30388950-66b50060-987f-11e7-83a3-ce29f6475262.png">

**Comment by brian-mann** [comment]
The code for this is merged, but it has not been released yet.

**Comment by brian-mann** [comment]
Fixed in [`0.20.1`](https://docs.cypress.io/guides/references/changelog.html#0-20-1).
